cookbooks/bcpc-hadoop/attributes/maintenance.rb

```diff
@@ -0,0 +1,10 @@
+default[:bcpc][:hadoop][:acl_group] = "acl_group_for_pam"
+default[:bcpc][:hadoop][:base_dn] = "DC=domain,DC=example,DC=com"
+default[:bcpc][:hadoop][:group_ou] = "OU=Groups,#{node[:bcpc][:hadoop][:base_dn]}"
+default[:bcpc][:hadoop][:domain] = "domain.example.com"
+default[:bcpc][:hadoop][:short_domain] = node[:bcpc][:hadoop][:domain].split('.')[0]
+default[:bcpc][:hadoop][:dir_threads] = 32
+default[:bcpc][:hadoop][:group_dir_mode] = "770"
+default[:bcpc][:hadoop][:user_dir_mode] = "755"
+default[:bcpc][:hadoop][:group_dir_prohibited_groups] = ['^users$', '^svn.*$', '^git.*$', 'dba']
```

Comment:
The purpose of this is not obvious from just the attribute name. A little comment would help. Also why `maintenance.rb` as the file name? Perhaps a comment as the first line can explain that?